config/routes.rb

```diff
@@ -3,6 +3,10 @@
     root 'articles#index'
 
     resources :articles
+    resources :users
+    
+    post "sign_up", to: "users#create"
+    get "sign_up", to: "users#new"
```

Comment:
figured /sign_up could be a more user friendly url than /users/new

Reply:
Yeah, another common option is "/register".

Reply:
`user/signup` or `user/register`?

Reply:
I'm not sure I follow the question.